src/resizer/src/Resizer.cc

```diff
@@ -2171,12 +2171,11 @@ Resizer::repairSetup(float slack_margin)
              delayAsString(worst_slack, sta_, 3));
   Slack prev_worst_slack = -INF;
   int pass = 1;
-  int decreasing_slack_passes = 0;
+  int decreasing_or_equal_slack_passes = 0;
   while (fuzzyLess(worst_slack, slack_margin)
-         // Allow slack to increase a few passes to get out of local minima.
-         && (decreasing_slack_passes < 0
-             || fuzzyGreater(worst_slack, prev_worst_slack))
-         && !fuzzyEqual(worst_slack, prev_worst_slack)) {
+         // Allow slack to increase or be equal a few passes to get out of local minima.
+         && (decreasing_or_equal_slack_passes < 5
```

Comment:
I don't ever put magic numbers in code like this. Thew 5 should be a static class variable with a name to say what it does.

Reply:
Makes sense, I'll incorporate that change